src/main/scala/cromwell/Main.scala

```diff
@@ -21,167 +20,184 @@ object Actions extends Enumeration {
-      println("usage: highlight <wdl> <html|console>")
-      System.exit(1)
+  def validate(args: Seq[String]): Int = {
+    continueIf(args.length == 1) {
+      loadWdl(args) { _ => exit(0) }
     }
+  }
 
-    val visitor = args match {
-      case a if a.length >= 2 && args(1) == "html" => HtmlSyntaxHighlighter
-      case _ => AnsiSyntaxHighlighter
+  def highlight(args: Seq[String]): Int = {
+    continueIf(args.length == 2 && Seq("html", "console").contains(args(1))) {
+      loadWdl(args) { namespace =>
+        val formatter = new SyntaxFormatter(if (args(1) == "html") HtmlSyntaxHighlighter else AnsiSyntaxHighlighter)
+        println(formatter.format(namespace))
+        exit(0)
+      }
     }
-    val namespace = WdlNamespace.load(new File(args(0)), WorkflowManagerActor.BackendType)
-    val formatter = new SyntaxFormatter(visitor)
-    println(formatter.format(namespace))
   }
 
-  def inputs(args: Array[String]): Unit = {
-    if (args.length != 1) usageAndExit()
-    try {
-      import cromwell.binding.types.WdlTypeJsonFormatter._
-      val namespace = WdlNamespace.load(new File(args(0)), WorkflowManagerActor.BackendType)
-      namespace match {
-        case x: NamespaceWithWorkflow => println(x.workflow.inputs.toJson.prettyPrint)
-        case _ => println("WDL does not have a local workflow")
+  def inputs(args: Seq[String]): Int = {
+    continueIf(args.length == 1) {
+      loadWdl(args) { namespace =>
+        import cromwell.binding.types.WdlTypeJsonFormatter._
+        namespace match {
+          case x: NamespaceWithWorkflow => println(x.workflow.inputs.toJson.prettyPrint)
+          case _ => println("WDL does not have a local workflow")
+        }
+        exit(0)
       }
-    } catch {
-      case e:SyntaxError => println(e)
     }
   }
 
-  def run(args: Array[String], workflowManagerSystem: WorkflowManagerSystem): Unit = {
-    if (args.length < 1 || args.length > 3) usageAndExit()
+  def run(args: Seq[String]): Int = {
+    continueIf(args.nonEmpty && args.length <= 4) {
+      // The path is our first argument.
+      val wdlPath = Paths.get(args.head)
+      // Other arguments may default to using the head with different extensions.
+      def swapWdl(ext: String) = swapExt(args.head, ".wdl", ext)
+
+      // Convert the arguments into a lifted Seq[Option[String]].
+      val argOptions = args.lift
+      // If we ask for an argument, we'll give a default. Also, "-" is the same as explicitly saying "skip".
+      def argPath(idx: Int, default: => Option[String]): Option[Path] =
+        argOptions(idx) orElse default filterNot (_ == "-") map wdlPath.resolveSibling
+
+
+      // Inputs will be assumed to be at ".json" (unless overridden like all paths with a "-")
+      val inputsPath = argPath(1, Option(swapWdl(".json")))
+      // We'll check if the ".options.json" file exists and use it
+      val workflowOptionsPath = argPath(2, Option(swapWdl(".options.json")).filter(f => Files.exists(Paths.get(f))))
```

Comment:
Personally I prefer `.inputs` for inputs and `.options` for options as they're both JSON;  how do others feel?